ui/app/rif/components/searchDomains.js

```diff
@@ -2,16 +2,34 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import rifActions from '../../rif/actions'
+import actions from '../../actions'
 
 class SearchDomains extends Component {
 	_handleKeyDown = (e) => {
 		if (e.key === 'Enter') {
+			let domain = e.target.value.toLowerCase()
+			//Theres a limitation in manager that domains with less 5 characters are blocked
+			if(e.target.value.toLowerCase().length <= 5){
+				this.props.displayWarning("Domains with less than 5 characters are blocked.")
+				return
+			}
+
 			let domains = JSON.parse(localStorage.rnsDomains);
 			let existDomain = domains.find(domain => domain.domain === e.target.value.toLowerCase())
 			if(existDomain)
 				return this.props.showDomainsDetailPage(existDomain)
-
-			//Here goes the logic to search domains that are not in localstorage
+			//Checks if the domain is available, so if it is, it need to render a screen so the user can register it
+			this.props.checkDomainAvailable(e.target.value.toLowerCase()).then(domainToRegister => {
```

Comment:
I think `domainToRegister` isnt clear. That variable name tells that we are trying to register a domain, but the generic case its just a search. Depending on the results the app goes to the registration or details. 

So i would name it just `domain` or something

Reply:
You right, i renamed this vars!